rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -33,34 +34,46 @@ def on_init(self, context):
         vars = context['vars']
 
         self.statement_length = vars.get(
-            'deoplete#sources#jedi#statement_length', 0
-        )
+            'deoplete#sources#jedi#statement_length', 0)
         self.use_short_types = vars.get(
-            'deoplete#sources#jedi#short_types', False
-        )
+            'deoplete#sources#jedi#short_types', False)
         self.show_docstring = vars.get(
-            'deoplete#sources#jedi#show_docstring', False
-        )
+            'deoplete#sources#jedi#show_docstring', False)
+        self.debug_server = vars.get(
+            'deoplete#sources#jedi#debug_server', False)
```

Comment:
What about using `True` here?
It is only used when generic debug is enabled, and then the server should be included probably by default?

I could imagine that `g:deoplete#sources#jedi#debug_server = 1` also enables generic debugging explicitly though - so we would have 3 states here..